bears/xml2/XMLBear.py

```diff
@@ -68,7 +91,7 @@ def process_output(self, output, filename, file):
                     output_regex=self._output_regex),
                 self.process_output_corrected(
                     stdout, filename, file,
-                    diff_severity=RESULT_SEVERITY.INFO,
+                    diff_severity=self._diff_severity,
```

Comment:
in praxis it's not (due to python executing everything in several processes), though this is a local variable and shall be used as such. So replace `self._diff_severity` with just `diff_severity` :+1: